tests/stockfish/test_models.py

```diff
@@ -296,28 +299,25 @@ def test_set_depth(self, stockfish):
         stockfish.get_best_move()
         assert "depth 12" in stockfish.info
 
-    def test_get_best_move_wrong_position(self):
+    def test_get_best_move_wrong_position(self, stockfish):
         wrong_fen = "3kk3/8/8/8/8/8/8/3KK3 w - - 0 0"
-        s = Stockfish()
-        s.set_fen_position(wrong_fen)
-        assert s.get_best_move() in (
+        stockfish.set_fen_position(wrong_fen)
+        assert stockfish.get_best_move() in (
             "d1e2",
             "d1c1",
         )
 
-    def test_get_parameters(self):
-        s1 = Stockfish()
-        s2 = Stockfish()
-        arg1 = s1.get_parameters()
-        arg2 = s2.get_parameters()
+    def test_get_parameters(self, stockfish):
+        arg1 = str(stockfish.get_parameters())
+        arg2 = str(stockfish.get_parameters())
         assert arg1 == arg2
```

Comment:
wrapping the parameters as a str is needed because a variable set to a dictionary will always reference the dictionary even after a key, value pair is updated.

Reply:
I think test case 

```
arg1 = str(stockfish.get_parameters())
arg2 = str(stockfish.get_parameters())
assert arg1 == arg2
```
doesn't make sense.

Maybe it would be better to return the previous version?

Reply:
You are right, this makes no sense. We can remove this part and leave in the get_parameters to check it works.